src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -137,6 +137,9 @@
     private Instant spanishContentLastUpdatedAt;
     private String scoringEngine = EMPTY;
     private String allowCalculator = EMPTY;
+    private Boolean aslUploadedPriorToLastContentUpdate;
+    private Boolean brailleUploadedPriorToLastContentUpdate;
+    private Boolean closedCaptioningUploadedPriorToLastContentUpdate;
```

Comment:
Not sure if I should set defaults here; if so I'd assume they should be set to `false`, which is the "happy path"; I think the normal case would be make sure the files are uploaded after the english content has been updated.

Reply:
Agree it should be primitive which will handle the default.